networkx/readwrite/json_graph/jit.py

```diff
@@ -0,0 +1,99 @@
+"""
+Read and write NetworkX graphs as JavaScript InfoVis Toolkit (JIT) format JSON.
+
+See the JIT documentation and examples at http://thejit.org
+
+Format
+------
+var json = [
+  {
+    "id": "aUniqueIdentifier",
+    "name": "usually a nodes name",
+    "data": {
+      "some key": "some value",
+      "some other key": "some other value"
+     },
+    "adjacencies": [
+    {
+      nodeTo:"aNodeId",
+      data: {} //put whatever you want here
+    },
+    'other adjacencies go here...'
+  },
+
+  'other nodes go here...'
+];
+"""
+
+#    Copyright (C) 2011-2013 by
+#    Aric Hagberg <hagberg@lanl.gov>
+#    Dan Schult <dschult@colgate.edu>
+#    Pieter Swart <swart@lanl.gov>
+#    All rights reserved.
+#    BSD license.
+
+__all__ = ['jit_graph', 'jit_data']
+
+import networkx as nx
```

Comment:
Imports from built-in libraries should appear before imports from the current project.